community-operators/vault/vaultoperator.v0.4.10.clusterserviceversion.yaml

```diff
@@ -0,0 +1,272 @@
+            "unsealConfig": {
+              "kubernetes": {
+                "secretNamespace": "default"
+              }
+            }
+
+            {
+              "config": {
+                "storage": {
+                  "file": {
+                    "path": "/vault/file"
+                  }
+                },
+                "listener": {
+                  "tcp": {
+                    "address": "0.0.0.0:8200",
+                    "tls_cert_file": "/vault/tls/server.crt",
+                    "tls_key_file": "/vault/tls/server.key"
+                  }
+                },
+                "telemetry": {
+                  "statsd_address": "localhost:9125"
+                },
+                "ui": true
+              }
+            }
+          }
+        }
+      ]
+spec:
+  displayName: Vault Operator
+  description: |
+    The Vault Operater creates and maintains highly-available Vault clusters on Kubernetes, allowing engineers to easily deploy and manage Vault clusters for their applications.
+
+    Secure, store and tightly control access to tokens, passwords, certificates, encryption keys for protecting secrets and other sensitive data using a UI, CLI, or HTTP API.
+
+    ### Reading and writing to Vault
+
+    Communicate with Vault through its command line utility `vault` via port forwarding:
+
+        $ kubectl --namespace default port-forward service/vault 8200:8200
+        $ export VAULT_TOKEN=$(kubectl get secrets vault-unseal-keys -o jsonpath={.data.vault-root} | base64 -D)
+        $ vault list secrets
+
+    The Vault-Operator helps to provision and manage Vault on top of Kubernetes with
+    the Bank-Vaults CLI tool which is a tool to help automate the setup and management of HashiCorp Vault.
+
+    Features:
+      - Initializes Vault and stores the root token and unseal keys in one of the followings
+        - AWS KMS keyring (backed by S3)
+        - Azure Key Vault
+        - Google Cloud KMS keyring (backed by GCS)
+        - Alibaba Cloud KMS (backed by OSS)
+        - Kubernetes Secrets (should be used only for development purposes)
+        - Dev Mode (useful for vault server -dev dev mode Vault servers)
+        - Files (backed by files, should be used only for development purposes)
+      - Automatically unseals Vault with these keys
+      - Continuously configures Vault with a YAML/JSON based external configuration (besides the standard Vault configuration)
+      - If the configuration is updated Vault will be reconfigured
+      - It supports configuring Vault secret engines, plugins, auth methods, and policies, audit devices
+      - Direct secret injection into Pods via the vault-secrets-webhook (https://github.com/banzaicloud/bank-vaults#mutating-webhook)
+
+  keywords: ['vault', 'secrets', 'security', 'banzaicloud', 'open source']
+  version: 0.4.10
+  maturity: alpha
+  replaces: vaultoperator.v0.4.10
```

Comment:
This implies that this CSV replaces itself. This field can be omitted as there are no previous versions of the CSV.

Reply:
Thank you, I have removed the field.